.github/workflows/pr-comments.yaml

```diff
@@ -0,0 +1,57 @@
+name: pull request comment
+on:
+  issue_comment:
+    types: [created, edited, deleted]
+env:
+  GH_USER: "github-actions[bot]"
+  GH_EMAIL: "<41898282+github-actions[bot]@users.noreply.github.com>"
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
It _appears_ the `issue_comment` workflow for comments in a fork PR in fact has access to a `GITHUB_TOKEN` with `write`.

I think we should not give the token to all the steps here.
While the workflow as defined in the default branch is used, it seems like a malicious actor could open a PR which executes arbitrary code when e.g. `make format` is run and they'd have access to a `GITHUB_TOKEN` with `write`.